samples/javascript_nodejs/language-generation/05.a.multi-turn-prompt-with-language-fallback/dialogs/userProfileDialog.js

```diff
@@ -48,8 +48,8 @@ class UserProfileDialog extends ComponentDialog {
 
         // Define per locale LG files.
         const templatesPerLocale = new Map();
-        templatesPerLocale.set('', Templates.parseFile(`${__dirname}/../Resources/UserProfileDialog.lg`)),
-        templatesPerLocale.set('fr', Templates.parseFile(`${__dirname}/../Resources/UserProfileDialog.fr-fr.lg`));
+        templatesPerLocale.set('', Templates.parseFile(`${ __dirname }/../Resources/UserProfileDialog.lg`));
+        templatesPerLocale.set('fr', Templates.parseFile(`${ __dirname }/../Resources/UserProfileDialog.fr-fr.lg`));
```

Comment:
We tend to not introduce this kind of changes, since they are not related to the code we are updating and can remain in inconsistent format with the rest of the code.

Reply:
There are a few more in the PR, we should leave them as they are to avoid noise in the required changes.